sc2/build/unix/build.config

```diff
@@ -80,7 +80,8 @@ uqm_requirements()
 			LDFLAGS="$LDFLAGS -s ASSERTIONS=0 -s ASYNCIFY -pthread"
 			# "Final" LDFLAGS only apply wheb building 'uqm', not config tests etc
 			LDFLAGS_FINAL="$LDFLAGS_FINAL --preload-file=content"
-			CCOMMONFLAGS="$CCOMMONFLAGS -s ASYNCIFY -pthread"
+			LDFLAGS_FINAL="$LDFLAGS_FINAL -lidbfs.js --pre-js wasm/pre.js"
+			CCOMMONFLAGS="$CCOMMONFLAGS -pthread"
```

Comment:
Is removal of `-s ASYNCIFY` here just a cleanup or did it cause some issue?

Reply:
It's an unrelated cleanup, putting ASINCIFY in CCOMOMONFLAGS is a noop (except a warning)